proto/buf.yaml

```
@@ -1,9 +1,11 @@
-# Generated by "buf config migrate-v1beta1". Edit as necessary, and
-# remove this comment when you're finished.
-#
-# This module represents the "proto" root found in
-# the previous configuration.
 version: v1
+name: buf.build/cosmos/ibc-go
+deps:
+  # see: (https://github.com/cosmos/cosmos-sdk/tree/main/proto#sdk-x-buf)
+  - buf.build/cosmos/cosmos-sdk:8cb30a2c4de74dc9bd8d260b1e75e176
+  - buf.build/cosmos/cosmos-proto:1935555c206d4afb9e94615dfd0fad31
+  - buf.build/cosmos/gogo-proto:34d970b699f84aa382f3c29773a60836
```

```suggestion
  - buf.build/cosmos/gogo-proto:bee5511075b7499da6178d9e4aaa628b
```

Must be pinned to this version.